app/js/controllers.js

```diff
@@ -0,0 +1,139 @@
+floripaApp.controller("GMapController", ['$scope', 'uiGmapGoogleMapApi', '$location', 'searchService', function ($scope, uiGmapGoogleMapApi, $location, searchService) {
+
+    $scope.goBack = function () {
+        $location.url('/');
+    };
+
+    $scope.search = function () {
+        searchService.streetName = $scope.streetName;
+        searchService.doSearch = true;
+        $location.url('/');
+    };
+
+    uiGmapGoogleMapApi.then(function (maps) {
+
+    });
+
+    $scope.putMarker = function (mapModel, location) {
+        if (!$scope.marker) {
+            $scope.marker = new google.maps.Marker({
+                position: location,
+                map: mapModel
+            });
+        } else {
+            $scope.marker.setPosition(location);
+        }
+    }
+
+    $scope.map = {
+        // should use the one from the device, just a mock
+        center: {
+            latitude: -27.5964796,
```

Comment:
How you would change this piece of code to make it clear which location is this?